.github/workflows/build-yocto.yml

```diff
@@ -7,18 +7,50 @@ on:
       - main
 
 jobs:
+  kas-lock:
+    runs-on: [self-hosted, x86]
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Run kas lock
+        run: |
+          kas dump --update --lock --inplace ci/base.yml
+
+      - name: Prepare artifact
+        run: |
+          mkdir ci/lock
+          mv -v ci/*.lock.yml ci/lock
```

Comment:
Why do you need to create a folder and move the files manually? You can set 'path' to ci/*.lock.yml directly in the upload-artifacts actions, no?

Reply:
no, when I tried first i use the the 'path' `ci/base.lock.yml` and the action create a empty directory on the artifacts.
but according to the docs https://github.com/actions/upload-artifact, what you suggest should be possible.
I probably did something wrong, I'll try again and if I could use this it would be much clearer.

Reply:
It works! thanks for pointing it.